cli/cmd/resources/instrumentor.go

```diff
@@ -463,7 +462,7 @@ func NewPodMutatingWebhookConfiguration(ns string) *admissionregistrationv1.Muta
 					},
 				},
 				FailurePolicy:      ptrGeneric(admissionregistrationv1.Ignore),
-				ReinvocationPolicy: ptrGeneric(admissionregistrationv1.IfNeededReinvocationPolicy),
+				ReinvocationPolicy: ptrGeneric(admissionregistrationv1.NeverReinvocationPolicy),
 				SideEffects:        ptrGeneric(admissionregistrationv1.SideEffectClassNone),
 				TimeoutSeconds:     intPtr(10),
```

Comment:
should this be configurable like in the helm chart?

Reply:
for cli, we have way less configuration options right now and there are many examples.
If something is really niche we support it in helm only.

Perhaps once #3482 landed all those very advanced options can be supported in cli with a `values.yaml` file